templates/Devel.dockerfile.j2

```diff
@@ -0,0 +1,115 @@
+# RAPIDS Dockerfile for {{ os }} "devel" image
+#
+# RAPIDS is built from-source and installed in the base conda environment. The
+# sources and toolchains to build RAPIDS are included in this image. RAPIDS
+# jupyter notebooks are also provided, as well as jupyterlab and all the
+# dependencies required to run them.
+#
+# Copyright (c) {{ now.year }}, NVIDIA CORPORATION.
+
+ARG CUDA_VERSION={{ DEFAULT_CUDA_VERSION }}
+ARG CUDA_MAJORMINOR_VERSION=${CUDA_VERSION}
+ARG LINUX_VERSION={{ os }}
+ARG PYTHON_VERSION={{ DEFAULT_PYTHON_VERSION }}
+
+FROM gpuci/miniconda-cuda-rapidsenv:${CUDA_VERSION}-devel-${LINUX_VERSION}-py${PYTHON_VERSION}
+
+{# devel environment config args #}
+ARG CC_VERSION=7
+ARG CXX_VERSION=7
+ARG PARALLEL_LEVEL=16
+
+ARG CUDA_MAJORMINOR_VERSION
+ARG PYTHON_VERSION
+
+ENV RAPIDS_DIR=/rapids
+
+{% if "ubuntu" in os %}
+ENV LD_LIBRARY_PATH=/opt/conda/envs/rapids/lib:${LD_LIBRARY_PATH}
+
+{# Update environment to use new gcc #}
+ENV CC=/usr/bin/gcc-${CC_VERSION}
+ENV CXX=/usr/bin/g++-${CXX_VERSION}
+ENV CUDAHOSTCXX=/usr/bin/g++-${CXX_VERSION}
+ENV PATH=${PATH}:/conda/bin
+{% endif %}
+
+{% if "centos" in os %}
+{# FIXME 20191002: remove this once gpuci base is rebuilt #}
+RUN yum -y update \
+  && yum -y install which patch \
+  && yum clean all
+
+{# Install gcc7 from prebuilt image #}
+COPY --from=gpuci/builds-gcc7:10.0-devel-centos7 /usr/local/gcc7 /usr/local/gcc7
+
+{# Update environment to use new gcc7 #}
+ENV GCC7_DIR=/usr/local/gcc7
+ENV CC=${GCC7_DIR}/bin/gcc
+ENV CXX=${GCC7_DIR}/bin/g++
+ENV PATH=${GCC7_DIR}/bin:$PATH
+ENV CUDAHOSTCXX=${GCC7_DIR}/bin/g++
+{# TODO-mwendt Not sure this should be restricted to just the GCC7_DIR #}
```

Comment:
Can this be addressed now?

Reply:
as discussed in our Slack call, we'll hold off on addressing these for now since they're outside the scope of this PR and the PR already includes a lot of changes.